lib/src/components/NxFileUpload/NxFileUpload.scss

```diff
@@ -47,6 +47,34 @@
   }
 }
 
+
+.nx-multi-file-upload {
+  width: 348px;
+
+  // need an enclosing div around NxMultiFileUpload that doesn't contain the validationError message
+  &__container {
+    $height: 220px;
+
+    min-height: $height;
+    box-sizing:border-box;
+    border: 1px solid var(--nx-swatch-indigo-90);
+    border-radius: 6px;
+
+    &__files {
+      padding: var(--nx-spacing-6x);
+      padding-bottom: 0px;
+      // total height of NxMultiFileUpload - total height of button with margin
+      max-height: calc(#{$height} - 90px);
+      box-sizing:border-box;
+      border-top: 1px solid var(--nx-swatch-indigo-90);
+    }
+
+    > .nx-btn {
+      margin: var(--nx-spacing-6x);
+    }
+  }
+}
```

Comment:
The selected files are supposed to have 8px of space in between

Reply:
> The selected files are supposed to have 8px of space in between

Can't believe I didn't notice that, thanks for pointing it out : [5f53d95](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/5f53d95466447b508a45864e290aa072e5b3a0bc)